Mathlib/NumberTheory/Ostrowski.lean

```diff
@@ -261,6 +285,23 @@ section Archimedean
 
 /-! ## Archimedean case -/
 
+/-- The usual absolute value on ℚ. -/
+def mulRingNorm_real : MulRingNorm ℚ :=
+{ toFun    := fun x : ℚ ↦ |x|
+  map_zero' := by simp only [Rat.cast_zero, abs_zero]
+  add_le'   := norm_add_le
+  neg'      := norm_neg
+  eq_zero_of_map_eq_zero' := by simp only [abs_eq_zero, Rat.cast_eq_zero, imp_self, forall_const]
+  map_one' := by simp only [Rat.cast_one, abs_one]
+  map_mul' := by
+    simp only [Rat.cast_mul]
+    exact_mod_cast abs_mul
+}
+
+@[simp] lemma mul_ring_norm_eq_abs (r : ℚ) : mulRingNorm_real r = |r| := by
+  simp only [Rat.cast_abs]
```

Comment:
Is this line necessary?

Reply:
It is used at some point in `mulRingNorm_equiv_standard_of_unbounded`